<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XML_Editor</class>
 <widget class="QMainWindow" name="XML_Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XML_Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionMinify"/>
    <addaction name="actionCheck_Consistency"/>
    <addaction name="actionFix_Errors"/>
    <addaction name="actionPrettify"/>
    <addaction name="actionConvert_To_JSON"/>
    <addaction name="actionCompress"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAdditional_Tools">
    <property name="title">
     <string>Additional Tools</string>
    </property>
    <addaction name="actionPost_Search"/>
    <addaction name="actionAnalyze"/>
    <addaction name="actionGraph"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Us"/>
   </widget>
   <widget class="QMenu" name="menuFont">
    <property name="title">
     <string>Font</string>
    </property>
    <addaction name="actionChoose_Font"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuAdditional_Tools"/>
   <addaction name="menuFont"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave_As"/>
   <addaction name="actionMinify"/>
   <addaction name="actionCheck_Consistency"/>
   <addaction name="actionShow_Errors"/>
   <addaction name="actionFix_Errors"/>
   <addaction name="actionPrettify"/>
   <addaction name="actionConvert_To_JSON"/>
   <addaction name="actionCompress"/>
   <addaction name="actionPost_Search"/>
   <addaction name="actionAnalyze"/>
   <addaction name="actionGraph"/>
   <addaction name="actionChoose_Font"/>
   <addaction name="actionAbout_Us"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/open-folder.png</normaloff>:/open-folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/save.png</normaloff>:/save.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout_Us">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/information-button.png</normaloff>:/information-button.png</iconset>
   </property>
   <property name="text">
    <string>About Us</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/close.png</normaloff>:/close.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionMinify">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/minimize.png</normaloff>:/minimize.png</iconset>
   </property>
   <property name="text">
    <string>Minify</string>
   </property>
  </action>
  <action name="actionCheck_Consistency">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/search.png</normaloff>:/search.png</iconset>
   </property>
   <property name="text">
    <string>Check Consistency</string>
   </property>
  </action>
  <action name="actionFix_Errors">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/repair.png</normaloff>:/repair.png</iconset>
   </property>
   <property name="text">
    <string>Fix Errors</string>
   </property>
  </action>
  <action name="actionPrettify">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/enhance-effect.png</normaloff>:/enhance-effect.png</iconset>
   </property>
   <property name="text">
    <string>Prettify</string>
   </property>
  </action>
  <action name="actionConvert_To_JSON">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/json.png</normaloff>:/json.png</iconset>
   </property>
   <property name="text">
    <string>Convert To JSON</string>
   </property>
  </action>
  <action name="actionCompress">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/compress.png</normaloff>:/compress.png</iconset>
   </property>
   <property name="text">
    <string>Compress</string>
   </property>
  </action>
  <action name="actionPost_Search">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/search (1).png</normaloff>:/search (1).png</iconset>
   </property>
   <property name="text">
    <string>Post Search</string>
   </property>
  </action>
  <action name="actionAnalyze">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/analyze.png</normaloff>:/analyze.png</iconset>
   </property>
   <property name="text">
    <string>Analyze</string>
   </property>
  </action>
  <action name="actionGraph">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/bar-chart.png</normaloff>:/bar-chart.png</iconset>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </action>
  <action name="actionChoose_Font">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/font-adjustment.png</normaloff>:/font-adjustment.png</iconset>
   </property>
   <property name="text">
    <string>Choose Font</string>
   </property>
  </action>
  <action name="actionShow_Errors">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/warning.png</normaloff>:/warning.png</iconset>
   </property>
   <property name="text">
    <string>Show Errors</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
